### PR TITLE
Fix/get item history

### PR DESCRIPTION
**Pull request by abdullahjankhan1**
## Description

Closes: #XXXX

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by faddat** [comment]
this looks important.

Can we get it in?

Marking ready for review to flag for attention

File: proto/pylons/pylons/item.proto [hunk 1]
```diff
@@ -44,5 +44,10 @@ message Item {
   string recipe_id = 15;
 }
 
-
```
Comment (catShaark) [inline proto/pylons/pylons/item.proto:47]: I think you should not include this change

Comment (catShaark) [reply proto/pylons/pylons/item.proto:47]: I mean this deletion

**Comment by nghuyenthevinh2000** [comment]
@abdullahjankhan1

Is it possible that you edit your pr to include picture of query result?

How can I replicate that result?

**Comment by abdullahjankhan1** [comment]
> @abdullahjankhan1
> 
> Is it possible that you edit your pr to include picture of query result?
> 
> How can I replicate that result?
> 
> Looking at the code doesn't give me a sense of what it is doing.

Yes sure,
Query Result Image
Response via LCD image

**Comment by abdullahjankhan1** [comment]
> @abdullahjankhan1
> 
> Is it possible that you edit your pr to include picture of query result?
> 
> How can I replicate that result?
> 
> Looking at the code doesn't give me a sense of what it is doing.

So the purpose here is to provide the ownership history of an NFT from when the item is first purchased and onwards.

**Review by faddat — APPROVED** [review]
thanks!

Will be useful as we move towards an ibc nft implementation that resembles mutexes!

**Pull request merged**